packages/instructors/.eslintrc

```diff
@@ -7,7 +7,8 @@
     "react/prefer-stateless-function": 0,
     "import/no-extraneous-dependencies": ["error", {
       "devDependencies": true
-    }]
+    }],
+    "no-underscore-dangle": [2, { "allow": ["_id"] }]
```

Comment:
Better to replace the '2' with the actual string

Reply:
Updated